charts/sentry-apps/README.md

```diff
@@ -25,6 +25,7 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 |-----|------|---------|-------------|
 | clickhouse.replicas | int | `1` | total number of clickhouse replicas |
 | config | object | See [values.yaml](https://github.com/sentry-kubernetes/charts/tree/develop/sentry) of upstream project | extra configuration for some config maps |
+| hooks | object | See [values.yaml](https://github.com/sentry-kubernetes/charts/tree/develop/sentry) of upstream project | configure hooks |
```

Comment:
The link for "values.yaml" actually directs to the directory containing this file in the upstream project. Is there a reason not to directly link to the actual file? If not, I would suggest doing so, since I would consider it to be slightly more intuitive and easier to use. (Just like it is done on line 36.)
```suggestion
| hooks | object | See [values.yaml](https://github.com/sentry-kubernetes/charts/blob/develop/sentry/values.yaml) of upstream project | configure hooks |
```
Note that the analogous change, if desirable, should then also be applied to line 27.